server/src/main/java/org/apache/uniffle/server/audit/ServerRPCAuditContext.java

```diff
@@ -122,4 +126,9 @@ public ServerRPCAuditContext setArgs(String args) {
     this.args = args;
     return this;
   }
+
+  public ServerRPCAuditContext setReturnValue(String returnValue) {
```

Comment:
Could we change `setReturnValue` to `withReturenValue`. I feel the setter should return void instead of a object.

Reply:
Done